src/olympia/devhub/tests/test_views.py

```diff
@@ -1492,7 +1492,7 @@ def test_lwt_submit_redirects_to_addon_submit(self):
         url = reverse('devhub.themes.submit')
         response = self.client.get(url, follow=True)
         self.assert3xx(
-            response, reverse('devhub.submit.distribution'), 301)
+            response, reverse('devhub.submit.distribution'), 302)
```

Comment:
Mmm, this is an odd change.  Was there a bug in our implementation?

Reply:
Yeah, there was. Our implementation was re-assigning `status_code` in a weird and very broken way which led afaik to testing the target status code incorrectly (or, well, not at all). The change does make sense though, we're using `django.shortcuts:redirect` which defaults to a `302` redirect if `permanent` is not applied.